### PR TITLE
Update AMP configuration documentation

### PR DESCRIPTION
**Pull request by ludop-usharesoft**
- The plugin can now be deployed by calling a script
- Ask to Restart the brooklyn server after the plugin installation when required
- Precise the change to make in the command in the case of a brooklyn server
- Change AMP/Brooklyn configuration command to avoid duplicated lines in the instruction is run several times

image

### CODE REVIEW

File: admin/en/source/pages/config/config-amp.rst [hunk 1]
```diff
@@ -51,11 +47,31 @@ To configure UForge to use the AMP instance:
...
+
+    1. Run the following command on the UForge UI node:
 
 		.. code-block:: shell
 
-			role create --name launcher --entitlement deployments_access
-			user role add --roles launcher --account someUser
+			$ /opt/UShareSoft/uforge/tools/update_scripts/deploy_brooklyn_plugin/deploy_brooklyn_plugin.sh
+
+    2. If the script output contains ``please restart the Brooklyn/AMP server``, then on the AMP server instance, run as root:
```
Comment (emuus) [inline admin/en/source/pages/config/config-amp.rst]: run the following command ...

File: admin/en/source/pages/config/config-amp.rst [hunk 2]
```diff
@@ -11,25 +11,21 @@ Once your UForge platform deployment is complete you can configure `Cloudsoft AM
 	* a configured user in AMP server that will be used by UForge to connect
 
 
-To configure AMP to be compatible with UForge:
+Configure AMP server with required option
```
Comment (emuus) [inline admin/en/source/pages/config/config-amp.rst]: Titles should have initial caps (ie Configuring AMP Server with Required Options)

File: admin/en/source/pages/config/config-amp.rst [hunk 3]
```diff
@@ -11,25 +11,21 @@ Once your UForge platform deployment is complete you can configure `Cloudsoft AM
...
 
-	3. Install the UForge Brooklyn plugin into AMP:
 
-		* Log in to the AMP Portal: https://[amp-instance-host]/brooklyn-ui-catalog/#!/
-		* On the catalog page, click ``Upload to catalog``
-		* Select and upload the UForge Brooklyn plugin file
-
-
-To configure UForge to use the AMP instance:
+Configure AMP access in UForge
```
Comment (emuus) [inline admin/en/source/pages/config/config-amp.rst]: Configuring AMP Access in UForge

File: admin/en/source/pages/config/config-amp.rst [hunk 4]
```diff
@@ -51,11 +47,31 @@ To configure UForge to use the AMP instance:
 
 			$ /opt/UShareSoft/uforge-client/bin/uforge_ui_update.sh
 
-	4. At this point the deployment feature should be available for the root user. In order to allow other users to use this feature, give them the entitlement ``deployments_access``. For more information on entitlements refer to :ref:`create-roles`.
+
+Install UForge Brooklyn Plugin into AMP catalog
```
Comment (emuus) [inline admin/en/source/pages/config/config-amp.rst]: Installing UForge Brooklyn Plugin in AMP Catalog

File: admin/en/source/pages/config/config-amp.rst [hunk 5]
```diff
@@ -51,11 +47,31 @@ To configure UForge to use the AMP instance:
...
+    2. If the script output contains ``please restart the Brooklyn/AMP server``, then on the AMP server instance, run as root:
+
+		.. code-block:: shell
 
+			$ systemctl restart amp
+
+        .. note:: On a Brooklyn server, replace ``amp`` by ``brooklyn`` in the command.
+
+
+Configure UForge users access rights
```
Comment (emuus) [inline admin/en/source/pages/config/config-amp.rst]: Configuring UForge User Access Rights

File: admin/en/source/pages/config/config-amp.rst (hunk 1 above)
Comment (ludop-usharesoft) [reply admin/en/source/pages/config/config-amp.rst]: Thanks for the review, I will do the modification for the titles.
For the sub-sections, I think we are in the case where these actions can be done separately even if on a fresh install they have to do all of them (at least the first 3, the 4th is just a tip for configuring access rights).
For example on a UForge version update, we may ask in the release note that the admin executes the "Installing UForge Brooklyn Plugin into AMP Catalog" part again in order to deploy a new version of the plugin to the already installed AMP server, but the other part won't be necessary.
If it's just the IP/hostname of the AMP server that changes, they will only need to re-execute the "Configuring AMP Access in UForge" paragraph.

**Pull request merged**